### PR TITLE
Styling for Podcast Metadata tab (and some help text)

### PR DESCRIPTION
**Pull request by brandonhundt**
- [x] Styling
- [ ] Help text - I tried to pull this in like Augury, but its missing something!!!

### CODE REVIEW

File: app/assets/stylesheets/card.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
 @import "shared/bootstrap-variables";
 
 .card {
+  margin-bottom: 1.5rem;
```
Comment (brandonhundt) [inline app/assets/stylesheets/card.scss:4]: this improves the spacing app wide for cards.

File: app/assets/stylesheets/shared/form.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // material style floating form labels
 .form-floating {
   > label {
-    color: $gray-700;
+    color: $gray-600;
```
Comment (brandonhundt) [inline app/assets/stylesheets/shared/form.scss:4]: This is intended to differentiate between placeholder text and filled in text.

File: app/assets/stylesheets/shared/tabs.scss [hunk 3]
```diff
@@ -143,3 +143,7 @@
     }
   }
 }
+
+.tab-content {
+  padding: 1.5rem;
+}
```
Comment (brandonhundt) [inline app/assets/stylesheets/shared/tabs.scss:149]: Needed some solid padding within the tab content.

File: app/javascript/controllers/tooltip_controller.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  connect() {
+    var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
+    var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
+      return new bootstrap.Tooltip(tooltipTriggerEl)
+    })
+  }
+}
```
Comment (brandonhundt) [inline app/javascript/controllers/tooltip_controller.js:10]: I copied this from Augury

**Review by cavis — APPROVED** [review]
👍 Onwards!

**Pull request merged**